edflow/args.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+
+
+def get_parser(**parser_kwargs):
+    def str2bool(v):
```

Comment:
i guess it is fine like this, but in `parse_unknown_args` we use `ast.literal_eval` to parse values. for consistency we could do the same here, but `True` and `False` are covered anyway. Both are fine by me, just let me know if you want to keep it this way or change it before I merge

Reply:
As discussen in #255 and #231 we should aim for unified behavior, i.e. I would like to change it. That said, once we implement the unified approach (in #231 or elswhere) this should get automatically fixed. Thus I would opt for merging like this.